deeppavlov/dataset_iterators/dstc2_ner_iterator.py

```diff
@@ -20,14 +20,14 @@
 
 from deeppavlov.core.commands.utils import expand_path
 from deeppavlov.core.common.registry import register
-from deeppavlov.core.data.dataset import Dataset
+from deeppavlov.core.data.dataset_iterator import BasicDatasetIterator
 from deeppavlov.core.data.utils import download
 
 logger = logging.getLogger(__name__)
 
 
-@register('dstc2_ner_dataset')
-class DstcNerDataset(Dataset):
+@register('dstc2_ner_iterator')
+class Dstc2NerDatasetIterator(BasicDatasetIterator):
```

Comment:
The class is dependent on DSTC2 data, should remain `Dstc2NerDatasetIterator`.

Reply:
Ok